outputs/http.go

```diff
@@ -0,0 +1,66 @@
+package outputs
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/aquasecurity/postee/layout"
+)
+
+type HTTPClient struct {
+	Name    string
+	Client  http.Client
+	URL     *url.URL
+	Method  string
+	Body    string
+	Headers map[string][]string
+}
+
+func (hc *HTTPClient) GetName() string {
+	return hc.Name
+}
+
+func (hc *HTTPClient) Init() error {
+	hc.Name = "HTTP Output"
+	return nil
+}
+
+func (hc HTTPClient) Send(m map[string]string) error {
+	hc.Headers["POSTEE_EVENT"] = []string{m["description"]} // preserve and transmit postee header
+
+	resp, err := hc.Client.Do(&http.Request{
+		Method: hc.Method,
+		URL:    hc.URL,
+		Header: hc.Headers,
+		Body:   io.NopCloser(strings.NewReader(hc.Body)),
+	})
+	if err != nil {
+		log.Println("error during HTTP Client execution: ", err.Error())
+		return err
+	}
+
+	b, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return fmt.Errorf("unable to read HTTP response: %s", err.Error())
+	}
+
+	if resp.StatusCode != http.StatusOK {
```

Comment:
should 204 cause error like this?

Reply:
Good catch - I updated it to handle 200-299 (which are OK) better c57ef590122ba6b73fe07d84e079352edbc4cf5a